spec/03-plugins/18-request-termination/01-schema_spec.lua

```diff
@@ -19,39 +19,39 @@ describe("Plugin: request-termination (schema)", function()
 
   describe("errors", function()
     it("status_code should only accept numbers", function()
-      local ok, err = v({status_code = "abcd"}, schema)
-      assert.same({status_code = "status_code is not a number"}, err)
+      local ok, error = v({status_code = "abcd"}, schema)
+      assert.same({status_code = "status_code is not a number"}, error)
```

Comment:
please do not use `error` as a variable name. The common name for errors is `err` because `error` is a built in Lua function (which will be shadowed by the local declaration as applied above)